client/state.go

```diff
@@ -240,11 +250,17 @@ func (c *GRPCClient) GetBulkState(ctx context.Context, store string, keys []stri
 	}
 
 	for _, r := range results.Items {
-		item := &StateItem{
-			Key:   r.Key,
-			Etag:  r.Etag,
-			Value: r.Data,
+		item := &BulkStateItem{
```

Comment:
Default of a string field is "" already so need for the If statement. In fact you can just set everything in the constructor

Reply:
OK. Set all the fields in construtor now.